src/reactor.cc

```diff
@@ -0,0 +1,375 @@
+#include "reactor.h"
+
+#include "boost/shared_ptr.hpp"
+
+namespace tricycle {
+
+Reactor::Reactor(cyclus::Context* ctx) : cyclus::Facility(ctx) {
+  // capacities set somewhat arbitrarily
+  fuel_tracker.Init({&tritium_storage}, 1000.0);
+  blanket_tracker.Init({&blanket}, 100000.0);
+}
+
+std::string Reactor::str() {
+  return Facility::str();
+}
```

Comment:
This is auto generated when you make a new cyclus agent, and since I didn't exactly know what purpose it served, I decided it was better not to touch it... If it's something I should delete I'm happy to do it though!

Reply:
It is autogenerated as a minimal stub.  If you want more than the minimum information printed about your agent when it is converted to a string, then you need to add it here.  

The first line `Facility::str()` ensures that this minimal amount is written. If you don't add anything else, I am 99% sure this is the same as removing it, as `Facility::str()` will be used if you don't override it yourself.

Reply:
Ahh, gotcha. I'll try taking it out and see if it breaks anything. Worse case it shouldn't be too hard to add back down the road.